src/snowflake/snowpark/dataframe.py

```diff
@@ -1841,16 +1854,12 @@ def natural_join(
             |5    |6    |NULL  |
             --------------------
             <BLANKLINE>
-
-        Args:
-            right: the other :class:`DataFrame` to join
-            join_type: The type of join ("inner", "full", "left", "right"). The default value is "left".
         """
-        join_type = join_type or "inner"
+        join_type = kwargs.get("join_type") or how
```

Comment:
Do we plan to eventually deprecate `join_type`?

Reply:
I don't think we need to do it currently, because join_type also makes sense. And we just use `how` to align with pyspark and pandas